pkg/rpcclient/wsclient_test.go

```diff
@@ -373,6 +373,19 @@ func TestWSExecutionVMStateCheck(t *testing.T) {
 	wsc.Close()
 }
 
+func TestWSNotificationNameCheck(t *testing.T) {
+	// Will answer successfully if request slips through.
+	srv := initTestServer(t, `{"jsonrpc": "2.0", "id": 1, "result": "55aaff00"}`)
+	wsc, err := NewWS(context.TODO(), httpURLtoWS(srv.URL), WSOptions{})
+	require.NoError(t, err)
+	wsc.getNextRequestID = getTestRequestID
+	require.NoError(t, wsc.Init())
+	filter := "notification_from_execution_with_long_name"
+	_, err = wsc.ReceiveExecutionNotifications(&neorpc.NotificationFilter{Name: &filter}, make(chan *state.ContainedNotificationEvent))
+	require.Error(t, err)
```

Comment:
We now have dedicated error, thus, it can be checked directly. Try `require.ErrorIs(t, err, neorpc.ErrInvalidSubscriptionFilter)`, and for `TestWSExecutionVMStateCheck` as far.